src/youtube-player/youtube-player.ts

```diff
@@ -115,6 +115,147 @@ export class YouTubePlayer implements AfterViewInit, OnDestroy {
     this._player.destroy();
     this._destroyed.emit();
   }
+
+  /** Proxied methods. */
```

Comment:
Could you add a JsDoc for each of these? It's fine if they're all just the same link to the full reference, e.g.
```ts
See https://developers.google.com/youtube/iframe_api_reference#Functions
```